packages/gatsby/src/utils/webpack.config.js

```diff
@@ -257,17 +257,7 @@ module.exports = async (program, directory, suppliedStage) => {
       rules.media(),
       rules.miscAssets(),
     ]
-    if (store.getState().themes.themes) {
```

Comment:
We need to keep this part of the plugin for now as I reverted full node_modules traversal from https://github.com/gatsbyjs/gatsby/pull/15270.

I'll remove this piece in a follow up PR where I traverse through plugins again.

Reply:
Good to remove once https://github.com/gatsbyjs/gatsby/pull/15284 is in